ui/src/pages/Logs/Logs.tsx

```diff
@@ -27,7 +27,7 @@ import {
   useTheme,
 } from '@mui/material';
 import ContainerIcon from '../../components/ContainerIcon/ContainerIcon';
-import SideBar from '../../components/SideBar/SideBar';
+import SideBar from '../../components/Sidebar/Sidebar';
```

Comment:
The previous PR added a breaking bug which you needed by fix by changing this line.

This is fine, but on a future PR we need to revert this change and change the folder name as well to `SideBar`